### PR TITLE
Add Single Sign-On section

### PR DESCRIPTION
**Pull request by Kreyren**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [-] Submit one item per pull request. This eases reviewing and speeds up inclusion.
	- Adding a new section with 3 items
- [X] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  `Demo` links should only be used for interactive demos, i.e. not video demonstrations.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [X] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [X] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [X] Additions are inserted preserving alphabetical order.
- [X] Additions are not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [awesome-analytics](https://github.com/onurakpolat/awesome-analytics), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/).
- [X] The `Language` tag is the main **server-side** requirement for the software. Don't include frameworks or specific dialects.
- [X] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [X] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [X] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [X] Any software project you are adding to the list is actively maintained.
- [X] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1379,6 +1379,13 @@ _Related: [Monitoring](#monitoring)_
 - [Yacy](https://yacy.net/en/index.html) - Peer based, decentralized search engine server. ([Demo](https://search.yacy.net/), [Source Code](https://github.com/yacy/yacy_search_server)) `GPL-2.0` `Java`
 
 
+### Single Sign-On (SSO)
+
+- [Kratos](https://www.ory.sh/kratos/?utm_source=github&utm_medium=AwesomeList&utm_campaign=AwesomeSelfHost) - Next-gen identity server with Ory-hardened authentication, MFA, FIDO2, profile management, identity schemas, social sign in, registration, account recovery, and IoT auth. Golang, headless, API-only - without templating or theming headaches. ([QuickStart](https://www.ory.sh/kratos/docs/quickstart/), [Source Code](https://github.com/ory/kratos)) `Apache-2.0` `Golang`
```
Comment (darkdragon-001) [inline README.md:1384]: Please remove tracking parameters (`utm_*`):

```suggestion
- [Kratos](https://www.ory.sh/kratos/) - Next-gen identity server with Ory-hardened authentication, MFA, FIDO2, profile management, identity schemas, social sign in, registration, account recovery, and IoT auth. Golang, headless, API-only - without templating or theming headaches. ([QuickStart](https://www.ory.sh/kratos/docs/quickstart/), [Source Code](https://github.com/ory/kratos)) `Apache-2.0` `Golang`
```

File: README.md [hunk 2]
```diff
@@ -1379,6 +1379,13 @@ _Related: [Monitoring](#monitoring)_
 - [Yacy](https://yacy.net/en/index.html) - Peer based, decentralized search engine server. ([Demo](https://search.yacy.net/), [Source Code](https://github.com/yacy/yacy_search_server)) `GPL-2.0` `Java`
 
 
+### Single Sign-On (SSO)
+
+- [Kratos](https://www.ory.sh/kratos/) - Next-gen identity server with Ory-hardened authentication, MFA, FIDO2, profile management, identity schemas, social sign in, registration, account recovery, and IoT auth. Golang, headless, API-only - without templating or theming headaches. ([QuickStart](https://www.ory.sh/kratos/docs/quickstart/), [Source Code](https://github.com/ory/kratos)) `Apache-2.0` `Golang`
+- [PWM](https://github.com/pwm-project/pwm) - An open source password self-service application for LDAP directories ([Source Code](https://github.com/pwm-project/pwm)) `Apache-2.0` `Java`
```
Comment (fbartels) [inline README.md:1385]: This one does not look like a tools for SSO, but simply a tool to change passwords in an existing ldap tree.

**Review by fbartels — CHANGES_REQUESTED** [review]

**Comment by nodiscc** [comment]
Blocked by https://github.com/awesome-selfhosted/awesome-selfhosted/issues/1038

File: README.md (hunk 2 above)
Comment (Kreyren) [reply README.md:1385]: Afaik it's multifunctional where i would describe it's main function as SSO, see the quickstart https://www.ory.sh/kratos/docs/quickstart/

Btw. sorrt for a long delay i didn't get notification about your message

**Comment by Kreyren** [comment]
> Blocked by #1038

How is that a blocker?

**Comment by nodiscc** [comment]
> How is that a blocker?

See https://github.com/awesome-selfhosted/awesome-selfhosted/issues/2543 about keycloak (applies to other items in the SSO section as well) -> blocked by https://github.com/awesome-selfhosted/awesome-selfhosted/issues/1038

**Comment by nodiscc** [comment]
Please resubmit it to https://github.com/awesome-foss/awesome-sysadmin#identity-management

**Pull request closed without merging**